deploy/nl/embeddings.yaml

```diff
@@ -1,30 +1,95 @@
-  model: ft_final_v20230717230459.all-MiniLM-L6-v2
-  model_type: LOCAL
-undata_ft:
-  store: MEMORY
-  embeddings: gs://datcom-nl-models/embeddings_undata_2024_03_20_11_01_12.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
-  model: ft_final_v20230717230459.all-MiniLM-L6-v2
-  model_type: LOCAL
-bio_ft:
-  store: MEMORY
-  embeddings: gs://datcom-nl-models/embeddings_bio_2024_03_19_16_39_03.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
-  model: ft_final_v20230717230459.all-MiniLM-L6-v2
-  model_type: LOCAL
-medium_lance_ft:
-  store: LANCEDB
-  embeddings: gs://datcom-nl-models-dev/lancedb/lancedb_embeddings_medium_2024_04_09_07_22_21.ft_final_v20230717230459.all-MiniLM-L6-v2
-  model: ft_final_v20230717230459.all-MiniLM-L6-v2
-  model_type: LOCAL
-medium_vertex_ft:
-  store: MEMORY
-  embeddings: gs://datcom-nl-models/embeddings_medium_2024_03_14_16_38_53.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
-  model: dc-all-minilm-l6-v2-model
-  model_type: VERTEXAI
\ No newline at end of file
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: 1
+
+indexes:
+  medium_ft:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_medium_2024_03_14_16_38_53.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  sdg_ft:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_sdg_2023_12_26_10_03_03.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  undata_ft:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_undata_2024_03_20_11_01_12.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  bio_ft:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_bio_2024_03_19_16_39_03.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  medium_lance_ft:
+    store: LANCEDB
+    embeddings: gs://datcom-nl-models-dev/lancedb/lancedb_embeddings_medium_2024_04_09_07_22_21.ft_final_v20230717230459.all-MiniLM-L6-v2
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  medium_vertex_ft:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_medium_2024_03_14_16_38_53.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: dc-all-minilm-l6-v2-model
+  dc-all-minilm-l6-v2:
```

Comment:
Do we need these?  Can we make the names inline with what exists?  (I'd say maybe drop in this PR?)

Reply:
sorry, which ones are you talking about, the vertex ai ones? & what do you mean by make the names inline with what exists?

Reply:
Yes, sorry, I meant the vertex AI ones which use vertex AI store.  And instead of model name "dc-all-minilm-l6-v2", more like "medium_vertexstore" etc.

Reply:
hm, should we keep the vertex ai ones though since we're trying to move all the vertex ai information into this file and this allows to test the VertexAIStore

Reply:
Then should we make the index names more aligned (eg. underscores etc)?

What is the state of the stores?

Reply:
updated names

Not sure the state of the stores, but they're the ones used by the eval tools, @shifucun?